packages/flutter_test/lib/src/window.dart

```diff
@@ -875,12 +875,17 @@ class TestFlutterView implements FlutterView {
 
   @override
   void render(Scene scene) {
-    _view.render(scene);
+    // Do not render the scene in the engine. The engine only observes one
+    // instance of FlutterView (the _view), and it is generally expected that
+    // the framework will render no more than one `Scene` per frame.
```

Comment:
This will stop sending all scenes to the engine - even the ones that belong to the implicit view since the test framework is wrapping the implicit view with a TestFlutterView. I would expect this to break a bunch of things: certain screenshot tests, `flutter run`ing a test and watching it run on device, etc.

I was expecting that we would only mock this out for `FakeView` subclasses that don't correspond to engine-side views? Normal `TestFlutterView` that to correspond to an engine-side view, likely need to continue calling this?

Reply:
Ah, I didn't know this. Should I create a `TestStubbedFlutterView` that does not send anything to the `FlutterView`? Then `FakeView` can inherit from that. Or actually, all `FakeViews` are the same, so I can just dedupe that code.

Reply:
Sounds reasonable. This could likely be a class we don't expose in our public API, right? If it just lives somewhere under `packages/flutter/test`?

Reply:
`packages/flutter/test` SGTM as a location.

Reply:
Done.